platform/windows/display_server_windows.cpp

```diff
@@ -30,23 +30,23 @@
 
 #include "display_server_windows.h"
 
+#include "os_windows.h"
+
 #include "core/io/marshalls.h"
-#include "core/math/geometry_2d.h"
 #include "main/main.h"
-#include "os_windows.h"
 #include "scene/resources/texture.h"
 
 #include <avrt.h>
 #include <dwmapi.h>
 
-#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
-#define DWMWA_USE_IMMERSIVE_DARK_MODE 20
-#endif
-
 #if defined(GLES3_ENABLED)
 #include "drivers/gles3/rasterizer_gles3.h"
 #endif
 
+#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
```

Comment:
This looks like an addendum to dwmapi.h in case it's an old version. I think we can have a rule for these cases, but it's not trivial because I guess situations can vary a lot. In any case, I'd suggest the include and the addendum not having anything interleaved (besides maybe blank lines?).

Reply:
Fixed.